cmd/vulcan-zap/manifest.toml

```diff
@@ -8,15 +8,18 @@ Timeout = 36000 # 10 hours. Expressed in seconds as an integer.
 # Ignored scanners for fingerprint:
 # 40018 - SQL Injection - Too many false positive results with variable resources.
 # Source: https://www.zaproxy.org/docs/alerts/
+# max_scan_duration and max_rule_duration are expressed minutes
+# max_scan_duration 9h to allow get results before the 36000 seconds 10h check timeout.
 Options = """{
     "depth": 2, 
     "active": true, 
     "min_score": 0, 
     "disabled_scanners": ["10062", "10003", "10108"],
     "ignored_fingerprint_scanners": ["40018"],
     "max_spider_duration": 0,
-    "max_scan_duration": 0, 
+    "max_scan_duration": 540,
     "max_rule_duration": 0,
+    "max_alerts_per_rule": 10,
```

Comment:
Does this mean that a rule will report at most 10 issues? If that's the case, we could potentially miss vulnerabilities, right?

Reply:
I understand this will cap the number of alerts a single rule can trigger. <https://github.com/zaproxy/zap-extensions/tree/main/addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules>, i.e. 
If a rule has been detected in 10 locations, it will stop evaluating that rule for the rest of the pages.
If that's the case I think this can be a good optimisation.

Reply:
If it is at the risk of missing out vulnerabilities, I think this should be discussed further. Maybe I'm missing some context about Vulcan SaaS. @julianvilas @manelmontilla wdyt?

Reply:
I don't know the context either in this case. @jesusfcr could you explain further the reason behind it?

Reply:
For sure, we should change the name of the option to make clear that it only applies to the passive scan. Something like `pscan_max_alerts_per_rule`. `max_alerts_per_rule` is quite misleading and, actually, I had just reviewed the PR and forgot how it was used. Now taking into account that it only applies to passive scan rules, I understand that many alerts can be potentially the same, but not necessarily. For instance, [InformationDisclosureSuspiciousCommentsScanRule](https://github.com/zaproxy/zap-extensions/blob/main/addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/InformationDisclosureSuspiciousCommentsScanRule.java) (I did not review all of them, this is just one rule I think could be detected multiple times without being the same problem).

Reply:
If we keep the option I agree with changing its name to something that mentions it applies only to passive scans, but in any case, from the context we have now I would not limit the number of alerts per rule by default, basically because the reason given by you @jroimartin; I prefer to not lose alerts than to have duplicates. Maybe what we can do is to leave the option but not enable it by default, as I can see this option to could be useful in the future in some repos using Lava.

Reply:
Reverted